pkg/dataplane/util/ansible_execution.go

```diff
@@ -63,9 +63,10 @@ func AnsibleExecution(
 	}
 
 	ansibleEE := EEJob{
-		Name:      executionName,
-		Namespace: deployment.GetNamespace(),
-		Labels:    labels,
+		Name:             executionName,
+		Namespace:        deployment.GetNamespace(),
+		Labels:           labels,
+		EnvConfigMapName: "openstack-aee-default-env",
```

Comment:
This would make the ConfigMap required wouldn't it?
Should we add optional:True here, https://github.com/openstack-k8s-operators/openstack-operator/blob/98fbd73bd2606479f25151a4d9a56282c092f1bb/pkg/dataplane/util/ansibleee.go#L236

Reply:
Yes that does make sense.